models/get-models.js

```diff
@@ -6,4 +6,29 @@ exports.fetchTopics = () => {
             const allTopics = res.rows
             return allTopics;
         })
+}
+
+exports.fetchAllArticles = () => {
+    return db.query(
+        `SELECT a.article_id, a.author, a.title, a.topic, a.created_at, a.votes, a.article_img_url, CAST(COUNT(c.article_id) AS INT) comment_count
+        FROM articles a 
+        LEFT JOIN comments c 
+        ON a.article_id = c.article_id
+        GROUP BY a.article_id
+        ORDER BY a.created_at DESC;`).then((res) => {
+        const allArticles = res.rows
+        return allArticles;
+    })
+}
+
+exports.fetchCommentsByArticleId = (article_id) => {
```

Comment:
I understand why you left this in from our helpdesk but for future all your code on one PR should just be pertaining to that ticket